src/game/GameLetter.vue

```diff
@@ -62,14 +62,14 @@ const labelState = computed(() => {
 
 <style scoped>
 input {
-  width: clamp(30px, min(15vw, 8vh), 80px);
```

Comment:
Would it also work to lower this one to 20px (`width: clamp(20px, min(15vw, 8vh), 80px);`) instead of the `grid` on `.row`?

The reason is that this tries to keep the height in mind, some landscape devices. Example:
PR: 
![image](https://user-images.githubusercontent.com/6643053/152657446-990808cf-6c51-4fe6-955d-0345909686d8.png)
main: 
![image](https://user-images.githubusercontent.com/6643053/152657455-15231864-f232-4a7c-80ae-ddb6a122216b.png)
Haven't tested enough on different devices, but I guess this can happen for tablets 🤔